src/components/Image/Image.scss

```diff
@@ -0,0 +1,10 @@
+.Image {
+  display: block;
+  width: 100%;
+  margin-top: 4000px;
```

Comment:
curious about this value

Reply:
Just a temporary value to test intersection observer.